Barotrauma/BarotraumaServer/ServerSource/Items/Components/ItemLabel.cs

```diff
@@ -39,13 +39,19 @@ public float TextScale
             set;
         }
 
-        [Serialize("0,0,0,0", IsPropertySaveable.Yes, description: "The amount of padding around the text in pixels (left,top,right,bottom).")]
+        [Editable(DecimalCount = 0, VectorComponentLabels = new string[] { "inputtype.left", "inputtype.up", "inputtype.right", "inputtype.down" }), Serialize("0,0,0,0", IsPropertySaveable.Yes, description: "The amount of padding around the text in pixels.")]
         public Vector4 Padding
         {
             get;
             set;
         }
 
+        [Editable, Serialize(Alignment.Center, IsPropertySaveable.Yes, description: "The alignment of the label's text.")]
+        public Alignment TextAlignment { get; set; }
+
+        [Editable, Serialize("UnscaledSmallFont", IsPropertySaveable.Yes, "The label's font.")]
+        public GUIFont Font { get; set; }
```

Comment:
It seems potentially error-prone to me that we'd maintain two sets of these properties. It would be very easy for someone to do the mistake of editing these in the client project without realizing the changes need to be done to the server project as well (because surely UI stuff like this is client-only, so there's no need to touch the server code?). This also introduces some unnecessary code duplication.

I think it'd be preferable to define these in the shared project.

Reply:
Done, all duplicated properties are now defined in the shared source.